test/etl.js

```diff
@@ -1 +1,106 @@
+          message:
+            // eslint-disable-next-line quotes
+            "The following fields are missing the required 'RESO.OData.Metadata.LookupName' annotation: \n" +
+            'CountyOrParish, StreetDirSuffix, City, StateOrProvince, StreetDirPrefix, StreetSuffix'
+        }
+      ],
+      errorLog: []
+    };
+    const result = await processCucumberJson({
+      cucumberReport: errorOnlyReport
+    });
+    assert.deepStrictEqual(result, expected);
+  });
+
+  it('should find an error when the raw report has both error and passed steps', async function () {
+    const expected = {
+      cucumberReport: [
+        {
+          stepName:
+            'RESO Lookups using String or String Collection data types MUST have the annotation "RESO.OData.Metadata.LookupName"',
+          message:
+            // eslint-disable-next-line quotes
+            "The following fields are missing the required 'RESO.OData.Metadata.LookupName' annotation: \n" +
+            'CountyOrParish, StreetDirSuffix, City, StateOrProvince, StreetDirPrefix, StreetSuffix'
+        }
+      ],
+      errorLog: []
+    };
+    const result = await processCucumberJson({
+      cucumberReport: errorAndPassed
+    });
+    assert.deepStrictEqual(result, expected);
+  });
+});
+
+describe('parseErrorLog', function () {
+  it('should parse a single line error log', function () {
+    const expected = [
+      {
+        testStep: 'LookupResource',
+        message:
+          // eslint-disable-next-line quotes
+          "The following fields are missing the required 'RESO.OData.Metadata.LookupName' annotation: \n" +
+          'CountyOrParish, StreetDirSuffix, City, StateOrProvince, StreetDirPrefix, StreetSuffix'
+      }
+    ];
+    const result = parseErrorLog(errorLog);
+    assert.deepStrictEqual(result, expected);
+  });
+
+  it('should parse multi-line error log', function () {
+    const expected = [
+      {
+        testStep: 'DataAvailability',
+        message:
+          'Request to https://retsapi.raprets.com/2/bari/RESO/OData/Resource?$filter=DateTimeStamp lt 2022-12-19T22:20:02.461Z&$orderby=DateTimeStamp desc&$top=100 failed! No response code was provided. Check commander.log for any errors...'
+      },
+      {
+        testStep: 'DataAvailability',
+        message:
+          'Request to https://retsapi.raprets.com/2/bari/RESO/OData/History?$filter=ModificationTimestamp lt 2022-12-19T22:21:05.499Z&$orderby=ModificationTimestamp desc&$top=100 failed! No response code was provided. Check commander.log for any errors...'
+      },
+      {
+        testStep: 'DataAvailability',
+        message:
+          'Request to https://retsapi.raprets.com/2/bari/RESO/OData/Offices?$filter=ModificationTimestamp lt 2022-12-19T22:23:08.573Z&$orderby=ModificationTimestamp desc&$top=100 failed! No response code was provided. Check commander.log for any errors...'
```

Comment:
In general, we should try and remove the provider's API info. We can use something like test.reso.org.